tsconfig.json

```diff
@@ -3,7 +3,8 @@
         "module": "commonjs",
         "strict": true,
         "target": "esnext",
-        "lib": ["es2020", "dom"]
+        "lib": ["es2020", "dom"],
+        "outDir": "compiler"
```

Comment:
Moved JS into another dir - having the JS in the same folder makes it harder to browse

Reply:
I would even suggest moving all the source and `tsconfig.json` into a `src` directory - my personal take is that it's hard to discern what I should be looking for.

Reply:
I don't disagree with ^ either

Reply:
Do tests work with this? They use a relative path to reference baselines/local/... and baselines/reference/... I don't know whether that will end up relative to the root where it's run or relative to compiler/test.js.

Reply:
I set up github actions so ... let's see what happens when I close/re-open (I can't remember if you have to merge from main)